src/ClientHandle.cpp

```diff
@@ -129,7 +129,6 @@ cClientHandle::~cClientHandle()
 			World->RemovePlayer(m_Player, true);  // Must be called before cPlayer::Destroy() as otherwise cChunk tries to delete the player, and then we do it again
 			m_Player->Destroy();
 		}
-		delete m_Player;
```

Comment:
You sure about this? In the current system, the ClientHandle manages the player pointer's life.